mentat/code_context.py

```diff
@@ -220,24 +205,29 @@ async def refresh(self, replace_paths: bool | None = False):
 
     async def display_context(self):
         stream = SESSION_STREAM.get()
+        git_root = GIT_ROOT.get()
 
         if self.files:
             await stream.send("Files included in context:", color="green")
         else:
             await stream.send("No files included in context.\n", color="red")
             await stream.send("Git project: ", color="green", end="")
-        await stream.send(self.config.git_root.name, color="blue")
+        await stream.send(git_root.name, color="blue")
         await _print_path_tree(
-            _build_path_tree(list(self.files.values()), self.config.git_root),
-            get_paths_with_git_diffs(self.config.git_root),
-            self.config.git_root,
+            _build_path_tree(list(self.files.values()), git_root),
+            get_paths_with_git_diffs(git_root),
+            git_root,
         )
         await self.diff_context.display_context()
 
     async def get_code_message(
-        self, model: str, code_file_manager: CodeFileManager, parser: Parser
+        self,
+        all_file_lines: dict[Path, list[str]],
```

Comment:
In the `context-features` pr, the code_file_manager gets passed from here all the way down to the `CodeFile._get_file_message`. We can just use CODE_FILE_MANAGER.get() in that function and save a lot of ink.